src/main/starlark/builtins_bzl/common/python/py_executable_bazel.bzl

```diff
@@ -370,6 +376,16 @@ def _create_zip_file(ctx, *, output, original_nonzip_executable, executable_for_
             return None
 
     manifest.add_all(runfiles.files, map_each = map_zip_runfiles, allow_closure = True)
+    manifest.add_all(
+        runfiles.symlinks,
+        map_each = lambda symlink: _map_zip_runfiles_symlink(
+            _ZIP_RUNFILES_DIRECTORY_NAME + "/" + workspace_name,
```

Comment:
Will this work under bzlmod? Based on our other convo, the materialized runfiles path should be `_main` instead of the actual workspace name, right?

Reply:
Pretty sure it does since with Bzlmod `_main` *is* the workspace name (`ctx.workspace_name`).

Reply:
Ah, right, forgot that. It's just in the repo mapping the user-given name is put.